core/core.go

```diff
@@ -269,6 +262,22 @@ func initNetworkID(networkID int, meshDB *meshdb.MeshDB) error {
 }
 
 func (app *App) Start(ctx context.Context) error {
```

Comment:
@fabioberger In a previous PR I mentioned that we want to avoid generic names like `Start` and `Stop`. I couldn't think of a name in this case that isn't overly verbose. Open to suggestions if you have any.